_includes/graphql/cart-object.md

```diff
@@ -1,6 +1,7 @@
 Attribute |  Data Type | Description
 --- | --- | ---
 `applied_coupon` | [`AppliedCoupon`][AppliedCoupon] | The `AppliedCoupon` object contains the `code` text attribute, which specifies the coupon code
+`applied_gift_cards` | [`AppliedGiftCard`][AppliedGiftCard] | The `AppliedGiftCard` object contains the `code` text attribute, which specifies the gift card code. `applied_gift_cards` is a Commerce-only attribute, defined in the GiftCardAccountGraphQl module
```

Comment:
We can probably say that the `AppliedGiftCard` object that for now contains only the code is made this way as a wrapper to be extensible by 3rd party that wants to add data to it.
This is not a very important piece of information, but people might wonder why do we wrap everything in objects where we could of just used a String

Reply:
After talking to @keharper we decided this is a bigger issue that warrants a new topic. I created the following ticket to track the work needed:
https://jira.corp.magento.com/browse/MAGEDOC-3855